packages/ui/src/App.svelte

```diff
@@ -78,6 +78,10 @@
     const remixHandler = async (e: MouseEvent) => {
       e.preventDefault();
       if ((e.target as Element)?.classList.contains('disabled')) return;
+
+      const printVersionedModule = import('@openzeppelin/wizard/print-versioned');
+      const { printContractVersioned } = await printVersionedModule;
```

Comment:
This should work, doesn't it?

```suggestion
      const { printContractVersioned } = await import('@openzeppelin/wizard/print-versioned');
```